### ISSUE TITLE
Suggestions

### ISSUE THREAD

**Issue by MrPatben8** [kokernutz/traktor-kontrol-screens#16]
If you have any ideas or suggestions drop them down bellow.

**Comment by MrPatben8** [comment]
Created this issue mainly because I wanted to suggest adding a clock with the system time to the screen. Because of the S8's screens I pretty much never look at my laptop, I just close the lid and forget about it. However if I'm playing a set with a tight schedule it's pretty important to keep track of the time, ideally without having to look at the laptop or having to take out my phone. I tried to implement it myself, but because of the way I had done it I would only update the time if a track was playing. I'd like to see if anyone has any idea of how this could be done (perhaps some sort of overlay since my implementation meant that one of the 9 track data elements was replaced for the clock and that came with it's own slew of issues). It's not the biggest issue in the world but I think it would be a nice addition.

**Comment by skymakai** [comment]
I wear a watch, and have my phone set up with a recorded set in case of any computer issues.  Personally, I think cluttering up both screens with system time is a bad idea.

**Comment by MrPatben8** [comment]
True, I guess this is quite a particular problem for me since I don't wear a watch :P

**Comment by kokernutz** [comment]
I think it should be possible to add it as one of the options for the deck header. I will take a look next week!

**Comment by MrPatben8** [comment]
Cool thanks. I'll keep trying to do it myself, but I've only first touched QML like 2 days ago so I don't really know what I'm doing :P 
Also is there any documentation for Traktor (as in a list of properties that can be changed via QML)?

**Comment by sandmankfmf** [comment]
Hey!
Is it possible to make bar markers on a S8 waveform screens? Phrase length bars is OK, but some times it will be nice to have bolder grid stripes on the wave form to mark bars and not only beats.
